### PR TITLE
Making Zine - Week 2 

### PR DESCRIPTION
**Pull request by evelinabe**
This is a project made when doing the Technigo bootcamp. The project was about to learn how to:
 -structure a webpage  using boxmodel
 -layout a webpage using flexbox
 -create a responsive webpage for mobile, desktop and tablet.

I began with a quick scetch of the page. My main problem was keeping focus during coding. I problably spent more time on "nice to have"-features than "have to have". I also second guessed myself from time to time, so have probably redone the navbar four times. 

In the finished project I used boxmodel, flexbox, media queries, parallax image and boxshadow transitioning on hover.  

If I had had more time, I would have tried to truncate the paragraph in the main news card. I also tried to take away the card's textbox on laptop view and have it shown over an overlay on hover without rearring the HTML. My guess is Javascript is needed for that, though.

In the best of worlds the cards also would have had an asymmetrical layout using CSS grid. And I would have renamed two more classes in the main-article divs.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+                                </a>
+                                <a href="https://github.com/evelinabe" class="social-media-link">
+                                        <i class="fab fa-github fa-2x"></i>
+                                </a>
+                                <a href="https://www.instagram.com/evelinabe_/" class="social-media-link">
+                                        <i class="fab fa-instagram fa-2x"></i>
+                                </a>
+                        </div>
+                </div>        
+        </footer>
```
Comment (bealun) [inline index.html:287]: Really good looking footer, both style wise and how you've built it in HTML. So cool to see 🏆

**Review by bealun — APPROVED** [review]
Fantastic job!! 👏 I'm so impressed with the work that you've put into this projects with everything from the parallax scroll, the sticky menu to the hover effects and the stylish footer. I've left some comments with feedback that will help you along the way as well as some clapping hands where you deserve an applaud. Keep it up!
You should feel very proud of your work and this site, remember to celebrate!! 🎉

File: index.html [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+        <script src="https://kit.fontawesome.com/ada55aea8c.js" crossorigin="anonymous"></script>        <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&family=Reenie+Beanie&display=swap" rel="stylesheet">
+        <title>Document</title>
+</head>
+
+<body>
+        <header>
+                <nav class="navbar">
+                        <div class="navbar-logo">
+                                <a href="#" class="logo-link">MZ</a>
+                        </div>
```
[the remark this answers is not in the document]
Comment (evelinabe) [reply index.html:16]: I thought I did, thought I kept it at 2 indentations but now it looks like four 🥴☺️

[the remark this answers is not in the document]
Comment (evelinabe) [reply index.html:16]: I thought I did, thought I kept it at 2 indentations but now it looks like four 🥴☺️

**Comment by evelinabe** [comment]
> Fantastic job!! 👏 I'm so impressed with the work that you've put into this projects with everything from the parallax scroll, the sticky menu to the hover effects and the stylish footer. I've left some comments with feedback that will help you along the way as well as some clapping hands where you deserve an applaud. Keep it up!
> You should feel very proud of your work and this site, remember to celebrate!! 🎉

Thank you so much, @bealun , I'm blown away by your review. Will celebrate by playing with the kids :-D

**Pull request closed without merging**